docs/developer-guide/maps-configuration.md

```diff
@@ -135,6 +135,145 @@ Map configuration also contains the following additional options:
 - `offsetTime` the end of a time range
 - `timelineData` timeline options
 - `selectedLayer` selected layer id; if not present time cursor will be unlocked
+- `mapViews` map views options
+
+### mapViews
```

Comment:
I should move the whole section `## Additional map configuration options` at the end of the page.
The priority is anyway on the layers.

Reply:
@offtherailz done